src/core/surface/call.c

```diff
@@ -205,8 +205,8 @@ struct grpc_call {
   /* Received call statuses from various sources */
   received_status status[STATUS_SOURCE_COUNT];
 
-  void *context[GRPC_CONTEXT_COUNT];
-  void (*destroy_context[GRPC_CONTEXT_COUNT])(void *);
+  /* Contexts for various subsystems (security, tracing, ...). */
+  grpc_call_context contexts[GRPC_CONTEXT_COUNT];
```

Comment:
The issue is that I need to set the context (with the destructor) from the filter on the server and the only thing I have access to there is the transport op.